### ISSUE TITLE
Customer headers strips the character 's'

### ISSUE THREAD

**Issue by HolyBugx** [hahwul/noir#132]
Hi,

Firstly, it is a great tool, thanks for open-sourcing.

The issue is that the `--with-headers` flag removes the character 's' in the header value.

`--with-headers 'Accept-Version:test'` ends up as `Accept-Version: tet`.

Best,
~ hb

**Comment by hahwul** [comment]
Hi @HolyBugx 
Thank you so much for reporting the issue. 
It seems there was a typo in the header handling. I will fix it right away! 😄

**Comment by hahwul** [comment]
## Test
![](https://github.com/hahwul/noir/assets/13212227/74156ea9-4da7-41c1-a98b-8b501ed5ab10)

**Comment by hahwul** [comment]
Hi @HolyBugx 
The bug you reported appears to have been fixed, and closing the issue. It has been addressed in [v0.9.1](https://github.com/hahwul/noir/releases/tag/v0.9.1), and you can upgrade using the following method. If it happens to reoccur, please feel free to reopen the issue :D

```bash
# If you use home brew
brew upgrade

# If you do not see the updated version, please run this command.
brew update; brew upgrade
```

```bash
# Install with source
git clone https://github.com/hahwul/noir; cd noir
shards install
shards build
```
